src/shared/utils/flattenChildren.js

```diff
@@ -54,7 +54,10 @@ function flattenSingleChildIntoContext(
  * children will not be included in the resulting object.
  * @return {!object} flattened children keyed by name.
  */
-function flattenChildren(children: ReactElement<any>, selfDebugID: number): ?{ [name: string]: ReactElement<any> } {
+function flattenChildren(
+  children: ReactElement<any>,
+  selfDebugID?: number,
```

Comment:
My understanding was:

- `selfDebugID?: number` means that the argument is optional but whenever it is set, it is always a number.
- `selfDebugID: ?number` means that there's always an argument but it can be null.

But, it looks like it doesn't work that way. I'll post in the flow group.

https://flowtype.org/try/#0PQKgBAAgZgNg9gdzCYAoVUCuA7AxgFwEs5swAvAUwCc4B5K27CgQSoHMAKADwH4AuMNkwBbAEbUAlAKFjqYAN6owYKhXyYqpLmAA+OsAEYADAG5UAXww4CxUhS4BDAjACejFu24CeM8VSmCIn4KSipqGlq6+sZmlqiUNPTurJwSZgl0DEwpHMZp6PZO+K7JnvmFzm7ZnnlmQA